### ISSUE TITLE
How to use as controlled component?

### ISSUE THREAD

**Issue by cwrichardson** [cschroeter/park-ui#30]
First, thanks for doing this! It's a great example of using Ark + Panda, plus a nice library!

I'm trying to use the select component inside of react-hook-form as a controlled component, and ... it's not working :)

TBH, it really just could be that I'm doing something wrong, but I can't quite figure out how to get the `ref` passed properly, and don't know if it's me, or park-ui, or panda, or ark :) .

You can see a working (i.e., failing) example, here: https://github.com/cwrichardson/panda-test

Basically, if I try and use the `Controller` component from react-hook-form, it wants to pass down a `ref`, and the `Select` component doesn't like it:

```
Warning: Function components cannot be given refs. Attempts to access this ref will fail. Did you mean to use React.forwardRef()?

Check the render method of `panda.(props) => {
  const [useSelectProps, { children }] = (0,_create_split_props_mjs__WEBPACK_IMPORTED_MODULE_2__.createSplitProps)()(props, [
    "closeOnSelect",
...
```

**Comment by cwrichardson** [comment]
Closing this. It seems Ark-UI `Select` doesn't let one forward a ref, and there was some discussion about how to handle integration with `react-hook-form` in this closed pr (https://github.com/chakra-ui/ark/issues/1159). I'll open a PR over there to either accept a ref, or improve the docs. Thanks!
